compiler/f8cstatic.hpp

```diff
@@ -64,8 +64,11 @@ const BaseTypeMap FieldSpec::_baseTypeMap
 	{"EXCHANGE", FieldTrait::ft_Exchange},
 	{"MONTHYEAR", FieldTrait::ft_MonthYear},
 	{"UTCTIMESTAMP", FieldTrait::ft_UTCTimestamp},
+	{"UTCTIMESTAMPUS", FieldTrait::ft_UTCTimestampUs},
 	{"UTCTIME", FieldTrait::ft_UTCTimeOnly},
+	{"UTCTIMEUS", FieldTrait::ft_UTCTimeOnlyUs},
 	{"UTCTIMEONLY", FieldTrait::ft_UTCTimeOnly},
+	{"UTCTIMEONLYUS", FieldTrait::ft_UTCTimeOnlyUs },
```

Comment:
I am famialar with the link above but many FIX engines do not support us and ns. They typically session level reject based on format. But there is rapid adoption ongoing right now largely driven by MiFID2.